Cargo.lock

```diff
@@ -74,7 +74,7 @@ name = "arena"
 version = "0.0.0"
 dependencies = [
  "rustc_data_structures",
- "smallvec 1.0.0",
+ "smallvec 1.4.0",
```

Comment:
Was the `smallvec` update accidental, or some of the updated unicode crates require it?

Reply:
It seems it come from `unicode-security` crate version update.